opencog/cython/PythonEval.cc

```diff
@@ -579,50 +585,87 @@ void PythonEval::addSysPath(std::string path)
     //    PyRun_SimpleString(("sys.path += ['" + path + "']").c_str());
 }
 
+
+const int ABSOLUTE_IMPORTS_ONLY = 0;
+
 /**
 * Add all the .py files in the given directory as modules to __main__ and
 * keep the references in a dictionary (this->modules)
 */
-void PythonEval::add_module_directory(const boost::filesystem::path &p)
+void PythonEval::add_module_directory(const boost::filesystem::path &directory)
```

Comment:
@linas I'm going to dig into the bug in PythonEval that relates to its interactions with the GenericShell etc. The bug where you can't enter multi-line functions from the shell.

Once I do that, I should know enough to be able to effect a reasonable separation into a Python API calling class and one that defers to it for handling only the evaluation. I'll also check into the way the scheme code does this at the moment as we probably want to keep the Python and Scheme implementations as similar as possible given the current design of shell and evaluator interactions.